src/styles/blocks.scss/about-us.scss

```diff
@@ -0,0 +1,67 @@
+.about-us {
+  display: flex;
+  flex-direction: column;
+
+  &__foto {
+    width: 100%;
+    height: 100%;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    max-width: 570px;
+    padding: 20px 5px;
+    text-align: center;
+    background-color: #fff;
+    transform: translateY(-50%);
+    box-shadow: 0 2px 0 rgba(26, 90, 76, 0.25);
+
+    @include onTable {
+      padding: 45px 55px 50px;
+    }
+    @include onDesktop {
+      padding: 45px 100px 50px;
+    }
+  }
+
+  &__exhibition {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 15px;
+
+    @include onTable {
+      justify-content: start;
+    }
+
+    &-link {
+      margin-right: 30px;
+      text-decoration: none;
+      font-weight: 500;
+      font-size: 16px;
+      color: #687480;
+    }
+
+    &-date {
+      font-weight: 700;
+      font-size: 16px;
+      color: #1a5a4c;
+    }
```

Comment:
Why. do you use such a nesting here `&-`?

You could nest modifiers, or elements, but there you have non of them.

Watch more about naming convention in BEM [in this video](https://mate.academy/learn/css-advanced/bem#/video/920)
Watch closely when you use '-' in naming.